test/cids.go

```diff
@@ -42,6 +42,8 @@ var (
 	TestInvalidPath1 = "/invalidkeytype/QmaNJ5acV31sx8jq626qTpAWW4DXKw34aGhx53dECLvXbY/"
 	TestInvalidPath2 = "/ipfs/invalidhash"
 	TestInvalidPath3 = "/ipfs/"
+
+	TestResolvedPath = "/ipfs/zb2rhiKhUepkTMw7oFfBUnChAN7ABAvg2hXUwmTBtZ6yxuabc"
```

Comment:
`"/ipfs/" + TestCidResolved` ?

Reply:
doesn't need a separate variable really as it's not used anywhere, just putting it in `ipfs_mock`